mne/decoding/ems.py

```diff
@@ -76,38 +165,36 @@ def compute_ems(epochs, conditions=None, picks=None, n_jobs=1, verbose=None):
                          len(conditions))
 
     ev = epochs.events[:, 2]
-    # special care to avoid path dependent mappings and orders
+    # Special care to avoid path dependent mappings and orders
     conditions = list(sorted(conditions))
     cond_idx = [np.where(ev == epochs.event_id[k])[0] for k in conditions]
 
     info = pick_info(epochs.info, picks)
     data = epochs.get_data()[:, picks]
 
     # Scale (z-score) the data by channel type
+    # XXX the z-scoring is applied outside the CV, which is not standard.
```

Comment:
ok, to be addressed in a different PR. This is just to make a sklearn compatible transformer